std/object/client_trust.go

```diff
@@ -40,22 +40,25 @@ func (c *Client) ValidateExt(args ndn.ValidateExtArgs) {
 		overrideName = args.OverrideName
 	}
 
+	// Default fetch function
```

Comment:
Should we fetch certificates by default?
The current Object client does not have a deadline system, which means it may be vulnerable to infinite fetching loop. Need to look into this.

Reply:
My old design in NTSchema is that the lifetime of a certificate fetching Interest is the original Interest's lifetime minus time elapsed till the send time, so that the validation will fail when the lifetime of original Interest for the Data packet is reached.

Reply:
Sorry, unsure what you mean? Currently this API is used by other code and thus I wanted to maintain the backwards compatibility, just added a new functionality of being able to override the fetch function. (Currently that other code expects ValidateExt to fetch recursively I believe)